app/models/accounting/transaction.rb

```diff
@@ -3,12 +3,17 @@
 # Table name: accounting_transactions
 #
 #  accounting_account_id :integer
+#  amount                :decimal(, )
 #  created_at            :datetime         not null
+#  description           :string
 #  id                    :integer          not null, primary key
+#  private_note          :string
 #  project_id            :integer
 #  qb_id                 :string           not null
 #  qb_transaction_type   :string           not null
 #  quickbooks_data       :json
+#  total                 :decimal(, )
+#  txn_date              :date
```

Comment:
The table does already exist in production. What would your suggestion be otherwise @hooverlunch ?